projects/igniteui-angular/src/lib/directives/for-of/for_of.directive.ts

```diff
@@ -1,1809 +1,1810 @@
-            }
-            newSizesCache[item.currentIndex + 1] = newSizesCache[item.currentIndex] + newHeightCache[item.currentIndex];
-            newHeight += newHeightCache[item.currentIndex];
-        });
-        this.heightCache = newHeightCache;
-        this.sizesCache = newSizesCache;
-        return newHeight;
-    }
-
-    protected addLastElem() {
-        let elemIndex = this.state.startIndex + this.state.chunkSize;
-        if (!this.isRemote && !this.igxForOf) {
-            return;
-        }
-
-        if (elemIndex >= this.igxForOf.length) {
-            elemIndex = this.igxForOf.length - this.state.chunkSize;
-        }
-        const input = this.igxForOf[elemIndex];
-        const embeddedView = this.dc.instance._vcr.createEmbeddedView(
-            this._template,
-            new IgxForOfContext<T>(input, this.getContextIndex(input), this.igxForOf.length)
-        );
-
-        this._embeddedViews.push(embeddedView);
-        this.state.chunkSize++;
-    }
-
-    protected _updateViews(prevChunkSize) {
-        if (this.igxForOf && this.igxForOf.length && this.dc) {
-            const embeddedViewCopy = Object.assign([], this._embeddedViews);
-            let startIndex;
-            let endIndex;
-            if (this.isRemote) {
-                startIndex = 0;
-                endIndex = this.igxForOf.length;
-            } else {
-                startIndex = this.getIndexAt(this.scrollPosition, this.sizesCache);
-                if (startIndex + this.state.chunkSize > this.igxForOf.length) {
-                    startIndex = this.igxForOf.length - this.state.chunkSize;
-                }
-                this.state.startIndex = startIndex;
-                endIndex = this.state.chunkSize + this.state.startIndex;
-            }
-
-            for (let i = startIndex; i < endIndex && this.igxForOf[i] !== undefined; i++) {
-                const embView = embeddedViewCopy.shift();
-                this.updateTemplateContext(embView.context, i);
-            }
-            if (prevChunkSize !== this.state.chunkSize) {
-                this.chunkLoad.emit(this.state);
-            }
-        }
-    }
-    protected _applyChanges() {
-        const prevChunkSize = this.state.chunkSize;
-        this.applyChunkSizeChange();
-        this._recalcScrollBarSize();
-        this._updateViews(prevChunkSize);
-    }
-
-    /**
-     * @hidden
-     */
-    protected _calcMaxChunkSize(): number {
-        if (this.igxForScrollOrientation === 'horizontal') {
-            if (this.syncService.isMaster(this)) {
-                return super._calcMaxChunkSize();
-            }
-            return this.syncService.chunkSize(this.igxForScrollOrientation);
-        } else {
-            return super._calcMaxChunkSize();
-        }
-
-    }
-}
-
-/**
- * @hidden
- */
+﻿/* eslint-disable @angular-eslint/no-conflicting-lifecycle */
+import { CommonModule, DOCUMENT, NgForOfContext } from '@angular/common';

```

Comment:
I'm not sure what has happened here and what was changed but I don't think we should merge this change since it would lose the blame.